internal/core/graphics/image/cache.rs

```diff
@@ -28,6 +28,8 @@ impl clru::WeightScale<ImageCacheKey, ImageInner> for ImageWeightInBytes {
             #[cfg(not(target_arch = "wasm32"))]
             ImageInner::BorrowedOpenGLTexture(..) => 0, // Assume storage in GPU memory
             ImageInner::NineSlice(nine) => self.weight(_key, &nine.0),
+            #[cfg(feature = "unstable-wgpu-24")]
+            ImageInner::WGPUTexture(..) => 0, // Assume storage in GPU memory
```

Comment:
But this still has a cost. 
Does that mean that if this is part of the cache, it can grow forever and keep the memory in the GPU growing?
Or does this kind of texture never end up in the cache anyway and this is actually `unreachable!` ?

Reply:
The imported image is not stored in our cache (the point of the cache is to be able to reload from disk if entries are evicted. I'll change the commit to reflect that in clearer wording.